### PR TITLE
add more information about deep link for expo projects

### PR DESCRIPTION
**Pull request by eriveltonelias**

### CODE REVIEW

File: docs/deep-linking.md [hunk 1]
```diff
@@ -40,12 +40,50 @@ const SimpleApp = createAppContainer(createStackNavigator({...}));
...
 const prefix = Platform.OS == 'android' ? 'mychat://mychat/' : 'mychat://';
+// in Expo projects you can do the following to get the URI prefix
+const prefix = Expo.Linking.makeUrl('/');
 
 const MainApp = () => <SimpleApp uriPrefix={prefix} />;
 ```
 
 ## Set up with Expo projects
 
+you need to specify a scheme for your app. You can register for a scheme in your `app.json` by adding a string under the scheme key:
```
Comment (brentvatne) [inline docs/deep-linking.md]: should capitalize the first word in the paragraph, so "You need" rather than "you need"

File: docs/deep-linking.md [hunk 2]
```diff
@@ -40,12 +40,50 @@ const SimpleApp = createAppContainer(createStackNavigator({...}));
 // on Android, the URI prefix typically contains a host in addition to scheme
 // on Android, note the required / (slash) at the end of the host property
 const prefix = Platform.OS == 'android' ? 'mychat://mychat/' : 'mychat://';
+// in Expo projects you can do the following to get the URI prefix
+const prefix = Expo.Linking.makeUrl('/');
```
Comment (brentvatne) [inline docs/deep-linking.md]: we might want to have an entirely separate section for expo projects / vanilla react native projects, and just repeat this entire code example in both sections but with the prefix line that is relevant to the platform. it's a bit cleaner when you can just pick the section relevant to you and follow that

File: docs/deep-linking.md (hunk 1 above)
Comment (eriveltonelias) [reply docs/deep-linking.md]: 🤦🏾‍♂️
sorry haha

File: docs/deep-linking.md (hunk 2 above)
Comment (eriveltonelias) [reply docs/deep-linking.md]: yes, thanks! I'll do that.

**Comment by eriveltonelias** [comment]
done :)

**Comment by brentvatne** [comment]
💯

**Comment by brentvatne** [comment]
added a bit more clarification on the expo stuff :)

**Comment by eriveltonelias** [comment]
cool! thanks for add that awesome clarification. :)

**Pull request merged**